index.d.ts

```diff
@@ -1,8 +1,15 @@
+interface Options {
+	type?: 'hex' | 'base64' | 'url-safe';
+	characters?: string;
```

Comment:
We can actually enforce the `type` and `characters` exclusivity in the types. Can you use https://github.com/sindresorhus/type-fest/blob/c6a3d8c2603e9d6b8f20edb0157faae15548cd5b/source/merge-exclusive.d.ts#L35 here? Just add it as a dependency.